### PR TITLE
Ci release notes 1.42

### PR DESCRIPTION
**Pull request by lkilpatrick-harness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

#### Fixed issues

- The Intelligence Baseline popup was re-running the pipeline even when the close button was clicked. This has been fixed so that the close button now dismisses the popup as expected. (CI-13779, ZD-68120)

- On pipeline failure at init step entire pod yaml/object was being logged as error log for debugging as a result if any environment variables in a container referencing secrets present were also getting logged in delegate logs. Issue is resolved by changing the log type from error to debug type to remove visibility and added an extra log without logging entire object. (CI-13785)

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: release-notes/continuous-integration.md [hunk 1]
```diff
@@ -23,6 +23,16 @@ These release notes describe recent changes to Harness Continuous Integration.
 
 ## August 2024
 
+### Version 1.42
+
+<!-- 2024-08-12 -->
+
+#### Fixed issues
+
+- The Intelligence Baseline popup was re-running the pipeline even when the close button was clicked. This has been fixed so that the close button now dismisses the popup as expected. (CI-13779, ZD-68120)
```
Comment (nofarblue) [inline release-notes/continuous-integration.md:32]: let's remove this please, this is beta FF

**Review by nofarblue — CHANGES_REQUESTED** [review]
see comment

**Comment by dewan-ahmed** [comment]
Please update the file front matter date.

File: release-notes/continuous-integration.md (hunk 1 above)
Comment (lkilpatrick-harness) [reply release-notes/continuous-integration.md]: I have removed it.

**Review by nofarblue — APPROVED** [review]

**Pull request merged**